server/model/feature_flags.go

```diff
@@ -111,6 +113,7 @@ func (f *FeatureFlags) SetDefaults() {
 	f.OnboardingTourTips = true
 	f.AppsSidebarCategory = false
 	f.CloudReverseTrial = false
+	f.NewUI = false
```

Comment:
i guess you would also need to add flag in `server/config/client.go` & `server/platform/services/telemetry/telemetry.go`

Reply:
Oh I wasn’t aware of those, we will add there as well.

Reply:
@M-ZubairAhmed Why do we need to add feature flag in `telemetry.go` and `client.go`

Reply:
trackConfig function is used to send daily telemetry.
while another file is to generate client config.

Reply:
@M-ZubairAhmed I had discussions with Guillermo, we don’t need telemetry. `client.go` already have https://github.com/mattermost/mattermost-server/blob/2d61ec0b1da9fff6daaf80b71002b6003546646a/server/config/client.go#L367